R/post_ts.R

```diff
@@ -109,99 +112,20 @@ post_ts = function(files, on_exists=c("stop", "skip", "replace", "merge"), verbo
     # tag item with our special identifiers. if the item already existed,
     # identifiers should be wiped out by a known SB quirk, so sleep to give time
     # for the files to be added and the identifiers to disappear so we can replace them
-    for(wait in 1:100) {
-      Sys.sleep(0.2)
-      if(nrow(sbtools::item_list_files(ts_id)) > 0 && is.null(item_get(ts_id)$identifiers)) break
-      if(wait==100) stop("identifiers didn't disappear and so can't be replaced; try again later with ",
-                        "repair_ts('", ts_path$var_src, "', '", ts_path$site_name, "')")
+    if(expect_id_loss) {
+      for(wait in 1:100) {
```

Comment:
looks like even `item_upload_create` isn't ready for prime time - see https://github.com/USGS-R/sbtools/issues/39